### ISSUE TITLE
On Android tablets, excalidraw stutters and I have to wait a long time for the keyboard to go away

### ISSUE THREAD

**Issue by devinat1** [zsviczian/obsidian-excalidraw-plugin#874]
On my samsung S7+ tablet, I am using excalidraw and when I accidentally click twice with my pen on the screen, the keyboard pops up, and the excalidraw zoom resets to 100%.
This is a huge issue since I have to wait 20+ seconds until the keyboard dissapears again and I am able to go back to writing.
It is very slow for the keyboard to show up and dissapear again.
On my ipad, this is not an issue since the keyboard doesn't seem to pop up (or it may be because I have the magic keyboard plugged in).
Nonetheless, it would be great if the double tap to open the text entry was disabled when in tablet mode (TM).

**Comment by zsviczian** [comment]
Please try to disable zoom to fit in Excalidraw plugin settings. You'll find the setting under the display setting `Zoom to fit on resize`

**Comment by devinat1** [comment]
That makes the situation better, but it remains the case that excalidraw is quite slow for drawing and stutters when I continuously draw for further than 5 minutes.

**Comment by zsviczian** [comment]
Yes, unfortunately, Excalidraw is designed more as an easy illustration tool, not so much as a handwritten note-taking tool. While you can jot down a few words and create quick sketches if you need to take extensive handwritten notes in meetings or class excalidraw is probably not the best choice - at least not in its current state.

**Comment by devinat1** [comment]
I see. In that case is there anything you would recommend to integrate with obsidian that allows better drawing capability? I would really like something that automatically syncs with my vault. Ipad notes would work but not sure how to sync that.

**Comment by zsviczian** [comment]
Consolidating pen / stylus / handwriting / drawing tablet support discussion into #2773.

Closing this as a duplicate so related requests stay centralized. Performance, latency, and handwriting-quality concerns across devices are now tracked there.
